scene/gui/tabs.cpp

```diff
@@ -630,6 +630,7 @@ int Tabs::get_tab_idx_at_point(const Point2 &p_point) const {
 
 void Tabs::set_tab_align(TabAlign p_align) {
 
+	ERR_FAIL_INDEX(p_align, 3);
```

Comment:
@Noshyaar It gets optimised at compile-time, so it can't affect performance, at all..